WebApplicationTests/UnitTest1.cs

```diff
@@ -0,0 +1,379 @@
+using Autofac.Extras.Moq;
+using health_clinic_class_diagram.Backend.Model.Survey;
+using Moq;
+using System;
+using System.Collections.Generic;
+using WebApplication.Backend.Repositorys;
+using WebApplication.Backend.Services;
+using WebApplication.Backend.Util;
+using Xunit;
+
+namespace WebApplicationTests
+{
+    public class UnitTest1
+    {
+            [Fact]
+            public void AddNewSurveyTest()
```

Comment:
test name convention, others have used the sentence notations so something like Adds_new_survey would be appropriate